R/lint.R

```diff
@@ -187,7 +187,7 @@ lint_dir <- function(path = ".", relative_path = TRUE, ..., exclusions = NULL,
     seq_len(length(files)),
     function(i) {
       file <- files[i]
-      file %in% names(exclusions) && exclusions[[file]] == Inf
+      file %in% names(exclusions) && is_excluded_file(exclusions[[file]])
```

Comment:
Any idea why that happens?
The function is defined in `R/exclude.R` which even comes first in the collate order...

Reply:
This looks like an error from the linter. Maybe a bug in the linter.